docs/content/doc/installation/setup.en-us.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Setup"
+slug: "setup"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "installation"
+    name: "Setup"
+    weight: 40
+    identifier: "setup"
+---
+
+# Setup
+
+The first time you access the Gitea web interface, the home page will appear. 
+Clicking on "Register" or "Sign In" in the upper right corner will take you to 
+the setup page, where you can configure the database type and associated 
+settings, general settings such as networking and locations of files, and 
+optional settings such as email.
+
+[![Screenshot of the setup page](/images/gitea-setup-page-1.png)](/images/gitea-setup-page-1.png)
+
+When you are done configuring Gitea, click the button labeled "Install Gitea" at
+the bottom of the page. This will write your settings to `app.ini`.
+
+After finishing the setup process, you will be directed to a page where you can 
+create a user account. The first registered user will automatically become an 
+administrator.
+
+
+## Database Settings
+
+Includes database type and other database-dependent settings. You can choose 
+from SQLite3, MySQL, PostgreSQL, and Microsoft SQL Server. (The default database 
+system is SQLite3.)
+
+To set up Gitea with SQLite3, you need to specify the path to the database 
+file. If you're running Gitea as a service, this must be an absolute path.
+**If the database file does not already exist, Gitea should be able to create it.**
+
+For MySQL, PostgreSQL, and Microsoft SQL Server, you need to specify the 
+server address, the database name, and the user credentials that Gitea will use 
+to access the database. **Make sure the user exists on the database server first.**
+
+## General Settings
+
+In this section, you can set the general attributes of the Gitea instance. For 
+more details, see the [configuration cheat sheet](/en-us/config-cheat-sheet/).
```

Comment:
Can you use the Hugo `relref` function here?

`[configuration cheat sheet]({{< relref "doc/advanced/config-cheat-sheet.en-us.md" >}})`